tests/test_info.py

```diff
@@ -6,10 +6,21 @@
 import pytest
 
 import openedx_webhooks.info
-from openedx_webhooks.info import get_orgs, is_internal_pull_request
+from openedx_webhooks.info import get_orgs, is_internal_pull_request, get_updated_person
+
+import os
```

Comment:
Imports from the stdlib should go in a section at the top.  1) stdlib, 2) third-party modules, 3) our own modules.